### DISCUSSION TITLE
[Feature Request] <A simple, fast ux upgrade for the portal>

### DISCUSSION THREAD

**Discussion by SurfWeb3** [zkSync-Community-Hub/zksync-developers#62] [Feature requests]
### Network

Both

### Provide a brief description of the feature you would like to see implemented.

Here is a simple, fast frontend fix, that would enhance the UX. portal dot zksync dot io has a network selection menu, that seems almost hidden, in the left corner of the screen. after hours of using various wallets, swaps, bridges, etc., and being a bit tired at the time, I didn't notice the menu at first, which wasted some time for me. usually, such menus are at the top of the page, where they are easy to quickly notice, instead of put in the lower left corner where it almost seems hidden. 

### Who does this feature request benefit the most (users, developers, protocol, etc.)? users, especially people who do many transactions per day

easier and more intuitive use of the portal

**Comment by githubdoramon** [comment]
This change is on pur pipeline @SurfWeb3 . Thanks for pointing that out

**Comment by threadaqged** [comment]
To enhance the UX of the zkSync portal, consider relocating the network selection menu to a more prominent location, such as the top of the page, for improved visibility and accessibility. This change can greatly benefit users, especially those who frequently interact with the platform, by making network switching more intuitive and efficient.

**Comment by adambold** [comment]
Adding a network selection menu to the top of the page on portal.zksync.io would improve user experience, especially for users who perform frequent transactions. This change would make network switching more intuitive and accessible, enhancing the overall usability of the portal.

**Comment by tathimustingjason** [comment]
Enhancing the network selection menu placement on the portal.zksync.io frontend to improve user experience.

**Comment by seimabirthcan** [comment]
This feature request benefits users the most by enhancing the user experience and making network selection more intuitive on the portal.zksync.io platform. Placing the network selection menu at a more prominent location, such as the top of the page, would improve usability, especially for users who conduct frequent transactions.

**Comment by dianocsemintrey** [comment]
The user is suggesting a simple frontend fix for the portal on the zkSync network that would enhance the user experience. The user notes that the network selection menu is located in the lower left corner of the screen and can be easily missed, causing frustration and wasting time. The user suggests moving the menu to the top of the page for easier and more intuitive use of the portal. This feature request would benefit users, especially those who perform many transactions per day. No additional details are provided.
